### ISSUE TITLE
Check that second word in Javadoc for test is 'can'

### ISSUE THREAD

**Issue by mkordas** [teamed/qulice#662]
According to [this article](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names) second word in Javadoc in every method annotated with `@Test` should be `can`.

Let's enforce that convention by creating check dedicated for that.

**Comment by krzyk** [comment]
@mkordas I don't think it is possible in all the cases, first one that comes to my mind is e.g. *Class can't accept null values.*.
I'll try to look for other cases that would need to be added as an exception to this rule.

**Comment by krzyk** [comment]
@mkordas OK, the only new rule I could find is the *can't*, if you agree, please add it to the description.

**Comment by krzyk** [comment]
@mkordas what do you think about the above? Will *can* be enough, are there cases where *can't* could help?

**Comment by mkordas** [comment]
@krzyk thanks for pinging. Yes, I think we should be a bit more flexible here. Classes have functionalities but sometimes they also have limitations, and then what you propose makes sense. I'd just like to use bit more formal `cannot`.

**Comment by krzyk** [comment]
@davvd valid bug

**Comment by davvd** [comment]
> @davvd valid bug 

@krzyk tag "bug" added

**Comment by davvd** [comment]
@mkordas thank you for the ticket reported, I topped your account for 15 mins, transaction `56bdac20dd526255ca0001b1`

**Comment by krzyk** [comment]
@davvd this is postponed

**Comment by davvd** [comment]
> @davvd this is postponed

@krzyk got it, "postponed" label here

**Comment by davvd** [comment]
> @davvd this is postponed

@krzyk OK, I will try to assign someone else

**Comment by yegor256** [comment]
@mkordas we don't have javadoc blocks in test methods anymore
